tracing-appender/src/lib.rs

```diff
@@ -0,0 +1,55 @@
+use chrono::{DateTime, Datelike, TimeZone, Timelike, Utc};
+
+pub mod file_appender;
+
+mod inner;
+mod worker;
+
+#[derive(Copy, Clone, Eq, PartialEq, Debug)]
```

Comment:
We probably shouldn't derive `Copy` for this, in case we want to add more complex fields in the future that are not trivially copyable. In that case, if we had to take away the `Copy` implementation, that would be a breaking change, and I don't think that users are constructing and passing around `Rotation`s enough that `Copy` has a meaningful ergonomic benefit.